core/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+description = "A library for error handling in Kafka Streams."
+
+plugins {
+    id("com.github.davidmc24.gradle.plugin.avro") version "1.2.1"
+}
+
+sourceSets {
+    test {
+        resources {
+            srcDirs("src/test/avro")
+        }
+    }
+}
+
+dependencies {
+    val kafkaVersion: String by project
+    api(group = "org.apache.kafka", name = "kafka-streams", version = kafkaVersion)
+    val avroVersion: String by project
+    api(group = "org.apache.avro", name = "avro", version = avroVersion)
```

Comment:
this can be implementation as it is only used internally in the ErrorUtil but is not part of any method signature